.circleci/config.yml

```
@@ -81,6 +81,29 @@ commands:
           event: success
           condition: on_success
 
+  run_test_byte_code:
+    description: "Run test byte code on a environment."
+    parameters:
+      os_type:
+        type: enum
+        enum: ["ubuntu", "osx"]
+    steps:
+      - run: mkdir test-cases/
+      - run: cd test-cases && ../scripts/isolate_tests.py ../test/
+      - run: cd test-cases && ../scripts/bytecodecompare/prepare_report.py ../build/solc/solc --interface standard-json --report-file ../bytecode-report-${<< parameters.os_type >>}-json.txt
```

Anything with bash variables in it should be quoted just in case.
```suggestion
      - run: cd test-cases && ../scripts/bytecodecompare/prepare_report.py ../build/solc/solc --interface standard-json --report-file "../bytecode-report-${<< parameters.os_type >>}-json.txt"
```